src/kibana/plugins/settings/sections/indices/_indexed_fields.html

```diff
@@ -3,3 +3,5 @@
   rows="rows"
   per-page="perPage">
 </paginated-table>
+
+<div ng-if="rows.length === 0">No fields.</div>
```

Comment:
A clearer message here would be helpful. Like *No matching fields found* perhaps. 

Also, some styling - padding, centered text, maybe even grey on grey?